src/movies.js

```diff
@@ -3,18 +3,59 @@
 
 
 // Get the average of all rates with 2 decimals 
+function ratesAverage (array) {
+	let averageRate = 0;
+	array.map((movie) => {
+		averageRate += movie.rate;
+	})
+	console.log(Math.round((averageRate/array.length) * 100) / 100)
+}
 
 
 // Get the average of Drama Movies
 
+function dramaMoviesRate (array) {
+	let averageRate = 0;
+	let count = 0;
+	array.map((movie) => {
+		movie.genre.map((genre) => {
+			if (genre === 'Drama') {
+				averageRate += movie.rate;
+				count++;
+			} 	
+		})
+		
+	})
+	console.log(Math.round((averageRate/count) * 100) / 100)
+}
+
 
 // Order by time duration, in growing order
 
+function orderByDuration (array) {
+	array.sort(function(a, b){return a.duration-b.duration});
+	console.log(array);
+}
 
 // How many movies did STEVEN SPIELBERG
 
+function howManyMovies (array) {
+	let count = 0;
+	array.map((movie) => {
```

Comment:
Nice way of doing this here :) could use a filter or forEach but I really like the idea to return the count :)